### PR TITLE
Add more irregular verbs

### PR DESCRIPTION
**Pull request by nerixim**
refs https://github.com/rossmeissl/verbs/issues/39

### CODE REVIEW

File: lib/verbs/conjugations.rb [hunk 1]
```diff
@@ -37,11 +37,44 @@
     verb.form :had,    tense: :past,    derivative: :participle
   end
 
+  conjugate.irregular :can do |verb|
```
Comment (nerixim) [inline lib/verbs/conjugations.rb:40]: maybe we need an option to let choose the regular can/canned/canned too?

**Comment by rossmeissl** [comment]
@nerixim any idea why the tests are failing?

**Comment by nerixim** [comment]
@rossmeissl 
It must be for the same reason as https://github.com/rossmeissl/verbs/pull/45/files#r544793197

**Review by rossmeissl — APPROVED** [review]
Thanks again @nerixim !

**Pull request merged**